lib/asciidoctor_plantuml/plantuml.rb

```diff
@@ -91,10 +91,16 @@ def plantuml_content_format(parent, code, format, attrs = {})
 
           # insert global plantuml config after first line
           config_path = parent.attr('plantuml-include', '', true)
-          begin
-            content = insert_config_to_content(parent, config_path, content, attrs) unless config_path.empty?
-          rescue StandardError => e
-            return plantuml_invalid_file(config_path, e.message, attrs)
+
+          unless config_path.empty?
+            begin
+              source_file = parent.document.normalize_system_path(config_path, nil, nil, recover: false)
```

Comment:
Can be simplified to:

```suggestion
              source_file = parent.document.normalize_system_path(config_path)
```